nailed.gemspec

```diff
@@ -1,3 +1,4 @@
+# OUTDATED, DO NOT USE ANYMORE!
```

Comment:
Hmm, then why keep the file if it is unused?
If it might be valuable for somebody, then add a deprecation date and drop the file at that date.

Otherwise, just drop the file - its not really gone - its still in git :)